TripKit/AddOns/Bookings/BookingKit/Controller/BPKBookingViewController.m

```diff
@@ -349,7 +349,10 @@ - (void)requestForm:(void(^)(void))completion
      
      if (error != nil) {
        // server error
-       [self handle:error];
+       [self handle:error defaultDismissHandler:^(UIAlertAction * _Nonnull action) {
+         [strongSelf.navigationController popViewControllerAnimated:YES];
```

Comment:
This is mainly used in the Transport > Public transport > Setup > Enter incorrect username/password.

The flow takes user to the next screen while waiting for the result. If error occurs, the user is left in that error screen and had to use the "back" button to go back to the screen where they can retry.